custom_components/biketrax/alarm_control_panel.py

```diff
@@ -101,4 +101,6 @@ async def async_alarm_arm_away(self, code: str | None = None) -> None:
     async def async_alarm_trigger(self, code: str | None = None) -> None:
         """Send alarm trigger command."""
         if not self.coordinator.read_only:
-            await self.device.set_stolen(True)
+            # There is not really a possibility to trigger an alarm, so at
+            # least enable it.
+            await self.device.set_guarded(True)
```

Comment:
maybe `set_guarded` could be a switch or a button attached to the device rather than the trigger command, as you rightfully commented, this doesn't trigger the alarm...

Reply:
The `async_alarm_trigger` is only executed when you want to trigger the alarm on command. The closest you can do is to enable the alarm then. 

What do you mean with a switch or button?